### DISCUSSION TITLE
Are you seeing these posts?

### DISCUSSION THREAD

**Discussion by DanHugoDanHugo** [InnovateForVegas/doc-org-discussions#3] [Team Posts]
Our recent house cleaning effort included moving All Members that are part of the CodeForVegas Github organization here, into a Team called _All Members_ , to which this post should be visible.

Under that team, are the __Active Members__ team and its subordinates per project. Hopefully this will allow some communications with those who are not quite keeping up with the day-to-day here.

If you have not been seeing these updates, that is probably a different thing that you won't even know about, so fret not!

**Comment by cory-simpson-dev** [comment]
I see it! I am excited to see the structure forming!

**Comment by DanHugoDanHugo** [comment]
Ah good. @cory-simpson-dev since you actually took the time to respond, FYI we have three website projects now with The Forkening, if you are still interested in jumping in there we have @noliveira95 just committing a first repo for the .org site (for the _business_ side, if you will), and then we need to have appealing portals for Coders and Creators… shout out if you want to join the fun!
